packages/design-system/README.md

```diff
@@ -51,16 +51,13 @@ The design system follows a variation of [ITCSS](http://thomasbyttebier.be/blog/
     ├── images
     ├── index.css       Compiled CSS
     ├── index.js        Compiled JSS
-    ├── index.scss      Uncompiled SCSS
-    ├── components      React component source, test & example files
+    ├── index.scss      SASS entrypoint
+    ├── components      React component JS files
     │   ├── Button
     │   └── etc...
-    ├── utilities       Utility classes for modifying individual CSS properties
-    │   ├── text-color
-    │   └── etc...
     └── styles
         ├── settings    Global variables, mixins, and functions
```

Comment:
Reformat order of File Structure `dist/` directories to ascending alphabetical order, example:
- move `components` directory to top of `dist/` sub-directory
- move `styles/settings` to second last of sub-directory of `styles/`

Reply:
I made some edits, but instead of alphabetical order, I tried to organize it in order of whats most important. Does that work instead?

Reply:
I was thinking readers might check this directory against the `/dist` file structure on their local system and get confused by it. But that works too, as it draws their attention to what's more important.